analyzer-comments/elm/feedback_request.md

```diff
@@ -0,0 +1,6 @@
+# feedback request
+
+If this automated feedback doesn't look right, please [open an issue in the `exercism/elm-analyzer` repository][elm-analyzer-issues].
+
+[elm-analyzer-issues]: https://github.com/exercism/elm-analyzer/issues?q=is%%3Aissue+is%%3Aopen+sort%%3Aupdated-desc
```

Comment:
I believe the link still needs to be updated: https://github.com/exercism/elm-analyzer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc insted of with the double `%`.

Reply:
It probably works with rendering, something is being escaped, but does not work as is.

Reply:
Yes, like I mentioned, the rendering will escape the link and it should work as expected.

Reply:
Thanks for mentioning that, I just a few minutes ago fixed a problem with a link with two dashes that I had to escape as well, for rendering.

Reply:
Well it didn't work in the end, the comment doesn't appear -_-